lib/repo_tools.pm

```diff
@@ -558,4 +559,20 @@ sub verify_software {
     return '';
 }
 
+=head2 ensure_ca_certificates_suse_installed
+    ensure_ca_certificates_suse_installed();
+
+This functions checks if ca-certificates-suse is installed and if it is not it adds the repository and installs it.
```

Comment:
I think you need to configure your editor to also properly wrap lines within comments :)

Reply:
I will do. I actually just moved this function from `lib/utils.pm` to `lib/repo_tools.pm`...